tools/ClassGen/NodeBuilder.cpp

```diff
@@ -213,6 +234,27 @@ void NodeBuilder::emitSettersGetters(std::ostream &os) const {
        << "() const { return getNthResult(" << idx << "); }\n";
     idx++;
   }
+  // -> variable outputs.
+  if (variableOutputMembers_.size()) {
+    std::string indexShift = std::to_string(nodeOutputs_.size());
+    for (const auto &op : variableOutputMembers_) {
+      os << "  const std::vector<NodeValue> get" << op.second << "() const {\n"
+         << "    unsigned_t indexShift = " << indexShift << ";\n"
+         << "    std::vector<NodeValue> valueList;\n"
+         << "    for (unsigned i = 0; i < " << op.second << "_size_; i++) {\n"
+         << "      valueList.push_back(getNthResult(indexShift + i));\n"
+         << "    }\n"
+         << "    return valueList;\n"
+         << "  }\n";
+      os << "  const NodeValue get" << op.second << "_Nth(unsigned i) const {\n"
```

Comment:
It could be useful to add a comment explaining the semantics. If I understand you correctly, this method would return the i-th element of the current variable size output, correct?

Reply:
Yes this is correct. We need such methods because there can be multiple variable outputs. the introduction of '_' is to better separate the output name (given by the user) from the functionality.

Reply:
Removed `_` to stick the the Glow naming convention